spec/services/token_manager_spec.rb

```diff
@@ -0,0 +1,46 @@
+require 'rails_helper'
+class Object
+  def headers
+    {"Authorization": "eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzUxMiJ9.
+      eyJ1c2VyIjoxLCJleHAiOjE0NTc5NjEyMzh9.
+      PTyASPDQ2BzMmktgnc57YXHEVPPiKS5lEg9xz2CgsUQMmtbptF3tnhvJ7YSLG3oX1HClKh"}
+  end
+end
```

Comment:
What do you think of making this token to be auto generated. This here is probably valid for ~24hrs (or thereabout); after which these tests would start failing.

Since (valid) token will be required for some other tests in future, I suggest you create a `method` in the support folder which would be executed to generated valid token to be used (and reused) for every other test.